admin-dev/themes/new-theme/scss/pages/_orders.scss

```diff
@@ -269,24 +269,15 @@ td .editProductInvoice {
 }
 
 .partial-refund {
-  display: none;
-
```

Comment:
@NeOMakinG your review would be helpful, since I remove css that you added in this commit:  https://github.com/PrestaShop/PrestaShop/commit/36a4549f7c42cbe0a0b3464b272c7a60c438a1f7

It was a PR about styleint and fixing display bugs in the BO:  #16763 

Something went wrong but I don't know what... maybe a bad conflict ? The css that I remove is specific to spartial refund anyway, so the risk of breaking something else is rather low.

Reply:
Well, that's maybe a miss from me on a rebase or something like this, anyway if this is required to make it work properly I've nothing to say :)